include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3142,6 +3150,163 @@ class FEFaceEvaluation : public FEEvaluationAccess<dim,
    * points is inaccurate and this value must be used instead.
    */
   const unsigned int n_q_points;
+
+
+private:
+  /**
+   * Return face number of each face of the current face batch.
+   */
+  std::array<unsigned int, VectorizedArrayType::size()>
+  compute_face_no_data()
+  {
+    std::array<unsigned int, VectorizedArrayType::size()> face_no_data;
+
+
+    if (this->dof_access_index ==
+        internal::MatrixFreeFunctions::DoFInfo::dof_access_cell)
+      {
+        if (this->is_interior_face == true)
```

Comment:
I think the check would be easier to read if this is merged with the last `else` case:
```suggestion
    if (this->dof_access_index !=
        internal::MatrixFreeFunctions::DoFInfo::dof_access_cell ||
        this->is_interior_face == true)
```